src/Input/ToggleSwitch/Styles.ts

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+import { ToggleBallProps, ToggleProps } from './ToggleSwitch.interface';
+
+const ToggleSwitchWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  position: relative;
+`;
+
+const Toggle = styled.div<ToggleProps>`
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  overflow: hidden;
+  position: relative;
+  transform: translate3d(0, 0, 0);
+  background-color: ${({
+    activeBackgroundColor,
+    inactiveBackgroundColor,
+    active,
+  }) => (active ? activeBackgroundColor : inactiveBackgroundColor)};
+  height: 25px;
+  width: 56px;
+  border-radius: 14px;
+`;
+
+const ToggleBall = styled.div<ToggleBallProps>`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 2;
```

Comment:
This seems like it can be safely removed since `z-index` can't work without setting the `position` of the element